docs/app/references/migration-guide.mdx

```diff
@@ -8,6 +8,51 @@ sidebar_label: 'Migration Guide'
 
 # Migration Guide
 
+## Migrating to Cypress 15.0
+
+This guide details the code changes needed to migrate to Cypress
+version 15.
+[See the full changelog for version v14.0](/app/references/changelog#15-0-0).
+
+### Node.js 20 and 22+ support
+
+Cypress comes bundled with its own
+[Node.js version](https://github.com/cypress-io/cypress/blob/develop/.node-version).
+However, installing the `cypress` npm package uses the Node.js version installed
+on your system.
+
+[See Node's release schedule](https://github.com/nodejs/Release). Node.js
+version 18 and 23 will no longer be supported when installing Cypress. The minimum Node.js
+version supported to install Cypress is Node.js 20 and 22+.
```

Comment:
```suggestion
[See Node's release schedule](https://github.com/nodejs/Release). Node.js
version 18 and 23 are no longer supported when installing Cypress. The supported Node.js
versions to install Cypress are now Node.js 20 and 22+.
```

You can't really have two minimums.
Node.js v24.0.0 was supposed be released today, however they are having compatibility issues with a newer V8 engine and the release is delayed. I'll drop a note if there is any new planning date available.

Reply:
good point. Once it releases we need to test Cypress out with it to see if anything is broken